.ddev/.setup/scripts/utils.sh

```
@@ -307,6 +551,138 @@ function create_symlinks_additional_extensions() {
+        if [ -f "${dir}ext_emconf.php" ]; then
+            ln -sr "$dir" "$target"
+        else
+            # Classic-mode package discovery only scans for ext_emconf.php
+            # (PackageManager::scanAvailablePackages), so a composer.json-only
+            # fixture like the default sitepackage would never be found. Link
+            # its contents individually into a real directory instead of
+            # symlinking the whole thing, so a generated ext_emconf.php can
+            # live alongside them without touching the fixture's own source
+            # tree (which is shared, unmodified, across every version slot).
+            mkdir -p "$target"
+            ( cd "$dir"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)